src/lib.rs

```diff
@@ -197,3 +203,19 @@ stun_codec::define_attribute_enums!(
         UseCandidate
     ]
 );
+
```

Comment:
This block feels different in intent than most of the other code in `lib.rs` which seems to currently be mostly about module setup. Is there somewhere else it could live?